Client/Application/AppDelegate.swift

```diff
@@ -68,14 +68,20 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
         self.window = UIWindow(frame: UIScreen.main.bounds)
         self.window!.backgroundColor = UIColor.Photon.White100
 
-        // Short circuit the app if we want to email logs from the debug menu
-        if DebugSettingsBundleOptions.launchIntoEmailComposer {
-            self.window?.rootViewController = UIViewController()
-            presentEmailComposerWithLogs()
-            return true
-        } else {
-            return startApplication(application, withLaunchOptions: launchOptions)
+        if DebugSettingsBundleOptions.saveLogsToDocuments {
+            if let defaultLogDirectoryPath = Logger.logFileDirectoryPath(inDocuments: false),
```

Comment:
We should move this into its own method just to keep the appDelegate more readable.